packages/types-algosdk/index.d.ts

```diff
@@ -109,20 +109,20 @@ export interface CompileOut {
 export interface Transaction {
   // fields copied from
   // https://github.com/algorand/js-algorand-sdk/blob/develop/src/transaction.js#L117
-  from: ParsedAddress;
-  to: ParsedAddress;
+  from: Address;
+  to: Address;
```

Comment:
Why we are changing this type? Why it was wrong before?

Reply:
`ParsedAddress` is represented as (in types)
```
export interface ParsedAddress {
  publicKey: string;
}
```
where as this should be a decoded address. (With `publickey` and `checksum`, both of type Uint8Array). Which was represented by type Address:
```
 export interface Address {
  publicKey: Uint8Array;
  checksum: Uint8Array;
}
```

Here is the screenshot from txn obj: 
![image](https://user-images.githubusercontent.com/33264364/101960457-7b043400-3c2d-11eb-8e50-a7f07afa15a0.png)

Reply:
thanks